api/src/network/controller/vote.ts

```diff
@@ -47,7 +47,7 @@ export async function voteForNetworkPermission(
     logger.debug("No action required.");
   }
 
-  return [200, { apiVersion: "1.0", data: "OK" }];
+  return [200, { apiVersion: "1.0", data: {} }];
```

Comment:
change it in schema too

Reply:
same story for network.approveNewOrganization and network.approveNewNodeForExistingOrganization

Reply:
changed data type to `object` in `schema.ts` for the following: 

- voteForPermission (line 3032)
- approveNewOrganization (line 3071) 
- approveNewNodeForExistingOrganization (line 31112)